core/test/unit/ghost_url_spec.js

```diff
@@ -1,12 +1,8 @@
 /* globals describe, beforeEach, afterEach, it */
-/*jshint expr:true*/
-var should     = require('should'),
```

Comment:
Again, `should` needs to be required in all test files. The only reason it works without is because it's already loaded & overridden Object.prototype. However if it's not included in each file, the files become interdependent and it becomes impossible to run a single test file with `grunt test:file-path` (which I do a looooot)